node/14/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM debian:buster-slim
+
```

Comment:
I removed the MAINTAINER clause because it is deprecated : https://docs.docker.com/engine/reference/builder/#maintainer-deprecated
Should we add a LABEL clause ? If yes, should I use the name of Damien like others images ?